plugins/topology/README.md

```diff
@@ -81,6 +81,59 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
 
   ```
 
+- The following must be added in`customResources` component in the [`app-config.yaml`](https://backstage.io/docs/features/kubernetes/configuration#configuring-kubernetes-clusters) file to view the edit code decorator clicking on which will take you to the Red Hat OpenShift Dev Spaces instance:
+
+  ```yaml
+   kubernetes:
+     ...
+     customResources:
+        - group: 'org.eclipse.che'
+          apiVersion: 'v2'
+          plural: 'checlusters'
+  ```
+
```

Comment:
@hmanwani-rh So the edit code decorator will redirect the user to Red Hat OpenShift Dev Spaces instance only when Red Hat OpenShift Dev Spaces operator is installed and configured on OCP cluster. If its not installed then it will redirect the user to the application git repo. I have added this point as a note.